src/main/java/de/adorsys/keycloak/config/service/ClientAuthorizationImportService.java

```diff
@@ -142,38 +142,70 @@ private void updateAuthorization(
             realmManagementPermissionsResolver.createFineGrantedPermissions(authorizationSettingsToImport);
         }
 
-        ResourceServerRepresentation existingAuthorization = clientRepository.getAuthorizationConfigById(
-                realmName, client.getId()
-        );
-
-        handleAuthorizationSettings(realmName, client, existingAuthorization, authorizationSettingsToImport);
```

Comment:
I don't see a replacement for this call. Do we still need something to handle [top-level properties](https://www.keycloak.org/docs-api/24.0.5/rest-api/index.html#ResourceServerRepresentation) like the `policyEnforcementMode` or `decisionStrategy`?